### ISSUE TITLE
Prettify stats output and get rid of -1 for fastest capture

### ISSUE THREAD

**Issue by pmakin** [teeworlds-instagib-elo/iCTFX#11]
Currently stats look pretty messy and boring, also they dont show whose stats this is
![image](https://user-images.githubusercontent.com/55580385/196768548-cf47d8e4-5d6d-4306-b246-22afadd5bd37.png)
Grouping some of them and making all message more standing out will happy everyone's eye
![image](https://user-images.githubusercontent.com/55580385/196769230-67a3d170-235f-4bd8-bcb2-a4a15000a96b.png)

The second 'issue' is fastest capture, i think it should be hidden even its not set yet, thats why setting it to 0 by default and print it out with other stats is a good idea, you can see that on previous ss

**Comment by hardliner66** [comment]
Agree, this needs some prettifying. Fastest Capture is currently hidden if there is no fastest capture, which is intended.

Maybe we should align the values as well:
```
Kills:                         5
Deaths:                    5
K/D:                         5
Shots:                      5
Hitrate:                    5
Captures:                 5
Captures per touch: 5
```

Maybe use dots instead of spaces. Also we should define what should be shown as percent and what not.
